content/github/using-git/adding-a-remote.md

```diff
@@ -43,6 +43,7 @@ $ git remote add origin https://{% data variables.command_line.codeblock %}/octo
 
 To fix this, you can
 
+* Use this command instead git remote set-url origin https://{% data variables.command_line.codeblock %}/octocat/Spoon-Knife
```

Comment:
Hi @gaurav-kabra-official 👋🏻  I'm a writer at GitHub.

I like your idea of adding details of how to change the URL for the remote. This seems like a great addition to the troubleshooting information. @ajs256 made a good suggestion for making this new tip easier to read by changing the command to `code`. 

It looks as if we already have a topic on how to change the URL for a remote: [Changing a remote's URL ](https://docs-1804--patch-2.herokuapp.com/en/free-pro-team@latest/github/using-git/changing-a-remotes-url). I'd recommend linking to that instead of including the command here. So something like:

```suggestion
* [Update the URL for the existing remote](/github/using-git/changing-a-remotes-url)
```